### PR TITLE
fix(navbar): fixed the height of navbar menu

### PR DESCRIPTION
**Pull request by nozomemein**
## 📝 Description

Modify the Navbar Menu modal styling when it's open.

## ⛳️ Current behavior (updates)

There's a slight gap between the bottom of the modal and the page.
Content behind the modal is visible through the gap.

## 🚀 New behavior

|before|after|
|--|--|
|スクリーンショット 2023-10-25 12 50 13|スクリーンショット 2023-10-25 12 49 47|

## 💣 Is this a breaking change (Yes/No):

No

## 📝 Additional Information

Nothing special👍

## Summary by CodeRabbit

- **Bug Fixes**
  - Improved height calculation for the navbar to better fit the viewport.
  - Corrected a typo in the menu height calculation when animation is disabled.

### CODE REVIEW

**Comment by RibeyrollesMatthieu** [comment]
Any update on this PR ? Right now I'm fixing it inside my own code, but would be great to have it fixed

**Comment by ryo-manba** [comment]
@nozomemein 
Sorry for the delayed review, and thank you for the PR.
Can you confirm if this behavior is still present in the latest version?
If it is, it would be helpful to have a link to the site where it's occurring.

**Comment by nozomemein** [comment]
@ryo-manba 
Thanks for the reply!!
As of today, the bug still exists in the [NextUI component page](https://nextui.org/docs/components/navbar).

Screenshot below.

スクリーンショット 2024-05-18 20 41 21

This bug is also reported in this issue.
- https://github.com/nextui-org/nextui/issues/2766

**Review by ryo-manba — APPROVED** [review]
Looks pretty good, Thanks for the PR!

File: packages/components/navbar/src/navbar-menu.tsx [hunk 1]
```diff
@@ -51,6 +51,7 @@ const NavbarMenu = forwardRef<"ul", NavbarMenuProps>((props, ref) => {
         style={{
           // @ts-expect-error
           "--navbar-height": height,
+          height: "calc(100vh - var(--navbar-height))",
```
[the remark this answers is not in the document]
Comment (nozomemein) [reply packages/components/navbar/src/navbar-menu.tsx]: Thanks for the review! I fixed it. 👍

**Review by jrgarciadev — APPROVED** [review]

**Pull request merged**